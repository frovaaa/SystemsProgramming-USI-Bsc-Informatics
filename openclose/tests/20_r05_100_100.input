{{{}}+-}+-`{{}[]}'<+{+{[]<>}{<>}<3{[`{`++`{{}<>{}[`']4<<{{}`{}5[0`'<{[][9]}>]'}>>}'--'}']}>-}->
{}{}[9{<{}`+0`'-[3]+-{+-[]`{<{}>}<[]64`{{6}[<<>>``{<[]{}{+-`7+-4`'3956`''}>}'']}'>'}'>}]
`{}<{`6{}'`4<<{{8`[+-+-8<[++`0+<{[]5+655{}[[[{[]{{}{}6}`{}{[]`{}'}'}]]]-}>-'--]>]'}}>>'}>'
[]+-[]+{{++`<{`+<+[<5`'{+7`{<{}>{}{{}}+[{`<[]{5{<><<4<{{}{{[<>]<>}}}>>>}}>'}]-}'-}>]->-'}>'--}}-
6{}[`'0{}`{{{{[`'{<`<{{}5{+-}{+{4{7{``''}8[[]<>]2[{<++-+[++`2[]{{}}'--]-->}]}}-}}>'>}]}}}}']
<>+{}<`[9+[][[<`'{[+-+-5]}<>>{{{{}`{{}}{1{<+[`']<<`[]'17{}>>-`'>`'1<>`[{}]'+{}-`'}}'}}}]]-]'>-
{}{`<{<{+-}[<{<`[+-`']{5<+`'{}->}'>{{{<6>{{}{}[[`{{<+-+`82+-{[][]`'3<>+<>-}'->}}']]}}}}}>]>}>'}
[+[]<<>>[4`[]{}'{`'{}}<[[9``'7`'{}{{`9{+-<{`{}'``{`'}'[{}]`{+-3[]`<{}<{[[]]}>>'}''}>}'}}']]>]-]
+{<>{9{}<>{{[]<<{<<>++-<{{{{{}}}[][6`+-{}{{}}']}{}}+<<`'>{<>9{{}}[[[{}[{+[]-}<>]]]]}>->->}>>}}}}-
{}<{`'47+{{`[]62{}<>{{[]+`'9`++4+{``<[`<>{{}{[]+-<[]{{+[+{{{<[]>}}}-]-}}>}}']>''}---'-}}'}}-}>
{}<7<<>>[+-{}{{9<{}<`+-1{`[<<7>+`'<+<<+[+[`[[]{{{}{{}`'}}}]'{{{+-+[]{`'}-}}}]-]->>->->]'}'>>}}]>
+-+-0{}+{}{}+-++-{`{[]{2{{}}`[]'[1{}4]}+-[{}{}[][][{`'{}}[+-`']]`']<<`{+-`{}'}'><{5[]}<>>>}'}--
`{}[{{{[+[{[{}]+`{`'[]{`'{}`<{``<5{}>{2{<><[][{{9+[{4<`+`{}{0+-}'-'>}]-}}]>}}''}>'}}'-}]-]}}}]'
[+-[+-9[+`<`{{`'{<{<0{}`[[]]{{}+{<{+-{`'+<[[[[<>+-{[{}]{+{[{{}}]+{}-}-}}]]]]>-}}>}-}'>}>}}}'>'-]]]
[]+<`'{+-{}}>-``+8`{}[]4'{}+[][[+-+-][{{[+-]{{[<><>{{+{[[+[{}7``+`'[]-'+++---']-]]}-}}]}}}}]]--''
[`{}[`'+`{{{[[+-8`'{[<<`'{}>>]`{}9{}'}{[`{+-`'`'1}[<>++6<`'>`[]+2+<{}0{`'}>--'--]']}]]}}}'-]']
<+-<<>5>{}`'<>><>`3<[+-+-`'`[4[<0+{<><`7{<[{}[[+`<><{[<>`<{++`'--{}`'1}>']}>'-]]]>}'>}->]]']>'
4[]++{}{}--<<9[5{{+[]<>{{}[]{{<+`[<><>]{{}{}`'}'->{}}<>{}<{}`+-[{+-5<{+-}>2[{+-{}}]}]'>}}-}}]>>
{}{{}69{{`'{}{1`{[]{{[+-<+<[]>-<{}[]48<>`{<6{<[]>}1>+<><>-}[7`[<{}>]'<`8+9-'>]'>>]}}}'}}}}
`<[`+-[<{{+3``'{{{`'{}2<`'{[{[[<{<<>[<<>`3<`[]'[+`<[{<++[+-{}]-->}]>'-]>'>]>}>]]}]}>}}}'-}}>]']>'
<><8+<+{}4[+`1[{}][[]]{[`+-[2{{<{{}}{}<[]>{+-1[]{{<+{}`'{{<{[]+-[6+[]-]}>}}->}}}>}}]']}'-]->->
`<2{}>``{{2+-<>}}76{`'+`++-[][<<{{{{+-}}<>[[<<{[2`+`[]3[+-{{}}]8{<{2}>}'-']}>>]]}}>>]-'-}'''
<{{}}>{<>}{+[]-1<[]>{}++-+`'8`'`'+-[]<`'58[]{{1<[`+[{}{[]<+->}<`{{`'}}{6<+`<>'->}'>]-']>}}>--}
<[]++`'<>[0{<>`'4{<{5}+{}->[``''{}<>+-`'<>]}{9}}<><+-[][{<>+<>-}{[+{}[{[{{+-}}]{+{}-}}]-]}]>]-->
++2{+-+-[]+[[]]`'{+++-+---<[{}[]]<1<{}>1[`++-`<`[6{+-{2{`{`{}'{}}{+{}{}-+-}'}`'}}]'>'-']>>}-}--
`9``''+6-'+-<`9<<{{{}[]`5'`{}<[<+`<>+`<>`'{}7<{}8<0`{}[{}```'{}'`'{+-}'<<>>]'>>'-'->]>'}}>>'>
`[{`6++--{}5'{}+-+-+`[]+-2<>{}'<{}>[]+[]+``'`{{{}8[{}][]4<>`+<>+{<7+[{}]->{6<>}}--'}}''---}]'
<<>>{}3[`'[{+-{<>{+[]{}`'2+0`'<>{}++--<{``4{}<<>+-<>{}>[+<+{{}}7`<><5>{8{7}}'->-]''}>--}}}]]
{}``'+-{`[]<{}<{}{[[<<``'4{}{}`<[]{}``+[][]-'+`{}{{}`5{[]{{}{{}}6`<{`<{}>'}>'}}'}'-'>''>>]]}>>'}'
{`'}<>{}+7``'{}'{1<><``{{}+-`'}{}{+-}`'<><{}><>''>+{0[{`{[]}'`[4{{<{}{+[[<`'{{}}>]]-}>}}]'}]}-}-
[]`'[+{{{}}}{<+`'{+`'<>-{}{<><>[``{}<<>`{{}}{}[]<[[]{{+<>-+{<++{[{{}[]}]<`'>`'}-->}-}}]>'>'']}}->}-]
<>``'{<>}[]<{[+<>++{[]<{+-}[[5{<>}{}+{{{{+5<<+7->3+-[`']>-[{`<`+-'{+-[][[]]}>'}]}}}}-]]>}---]}>'
<>{}{+-{{[+-]<<<+`'-<>5`'{[+{+-}[+8{<>}<>-[+<[]3{{`'+<{}>{{<+-+9{{[][{}+-]}}->}}-}}>-]]-]}>>>}}}
{3[++--{}+[]<>`'+<>-++<{6{}<+`[+<>[[][[`<`[9{{7}}+2+-{{{+-}{{}}}{+-}`{+-}'}-]'>']]]-]'->}>---]}
<[`']++-<[{}]+{[6{}+[+-{+4-}]-6[]]}+[{}5+4++++--++-{}`'{{}{{}7[]{{{<>+-}{}`<>+-'}}}}----]-->->
3`'{<{<>}>}<[`<>++`'--{}[[[`'][{{{[{+{+[`'<<[<<9`'+-[{}]>{}>`[0{+8{{}{+-}}-}]']>>]-}-}]}}}]]]']>
+`6[+`{{}2}'-]{}[{}`{`'{}}<><>[`'{<+-1{{`'5+{}{}-<+[+<>+-[]+`[]+{}`{}<2{{{}}}>'-'--]->}}>}]']'-
{+-+{}-}{}{`'{<<>7`[]+`+-`+<`'>-'{`+`'{[`'[]]`'}{[<`{<{2`+[{}]-'{}{}<{}{<+`'<>->}>}>}'>]}-'}'-'>}}
+`1`'`'<1<{<`{`<><+-{<`7{+`{}'-}<{`'5{<{+-[]{``'{{{9{[`[[<>]`'<<<>>>]']}}}}'}}>}}>'>}>'}'>}>>'-
<>`[[]]{{{}+-}{<+->}}``''+-{4[`{++--+-[]}`'+`<+`{+<++-{}-[1+`'2{}+`'--+-{<[]>{<>+-}}]>-}'->'-']}'
[<>]+-{{}}<><+-[`[{}]<+-+-{{}`[+-]<>'[+<8{<{+[+1{{`{}'}9{`++`<>3[]'-[]`'-'}+{[]}-}-]-}>}>-]}>']>
[{{}6{{[]3[+-][]`'[]}6+-[{}<`<<6[<{}{+-2[{}]}[[]]{<{}`'4>}{}`'>{{}{+-<>{+[`<>'7]-}}}]>>'>]}}]
+9{}{}{{`[{}[+[+<>-<`'[[``6{{{<>}<<{}+{<>{7<>{{++{}[]`'<[]>--[{}<>]}{[]}6`'}}}->>}}'']]>]-]]'}}-
+`{{}}{}`+-[]96``+{}-+[[<`[{[][]{{}{<`'><>{}{`'+{[{}{0{}[[+-{1<>}`[+{{}}-]']]}]}-}}}}]'>]]-''''-
<>{}{}[]{{<>`{}{{}[+`'{{<>{`[][+`{[{}+-]<<>>}{<<`<<><[`'<>]+-{[]{}{{4`[]'[]{}}}}>>'>>}'-]'}}}-]}'}}
[[]]5[[<++[<>{{+-}{{}}{[]{0{{}{}}7{{+[]{5{}`'+[]`+[]`'-'-[]<`[87<<+-<[[2]]>>>]'>}-}}}}}]-->]]
`{}{<>}'<{{}}[[]]{[[]][`']++-{<{+-+`<<+[4{[{}+<+3{{{<++{`'+<<>>-}-``{{<>}}''->}}}->-]}]->>'-}>}-}>
[]{+-```'{`0'}[]2{<`'{<[{[1[[<>]<>{}`{{{4<>+-{<9<+[[]{}+[]<>{{<<>3<{}>>}}-]->>}}}}']]}]>}>}''}
`{}'{{<><>}}<+[[<<>>{}{`'{<>{}+{+[+-]`'-}-<{{<[[]`[++`[`{}+-[<`'<[`[]']<{}[{}]>>>]']'--]']>}}>}}]]->
{`'}{+-}{{+[[]<{8<`'2+-[]+<[{}`[]<``+<><3{<``+-<8`'[]3{<{<`+{{[1]}}-'>}>}>''>}>-''>']>->}>]-}}
<[]+[]<>->+-[+{2+`'-{`9{[4<>{}]{7{}}<{{{`{<{<[][]>}>`{<><>5+-`<`+<<>>{{{{}}}}-'>'}'}'}}}>}'}}-]
<`<>[[<5<+-[[{+`'-{[]``+{<[<>]+<><<>`+-'>-[`{{}}{}{[]`'`'[]{}[+-{}]}0'+{[3{}<>]}-]>}-''}}]]>>]]'>
{[827[+`+-+-<[[]++{{}}[]-{{`+-'`'<{+{`'[]}[{}]<<{`'}>+<>{`'+[]{}-}<>-{`<7``+-''>'}>-}>}}-]>'-]]}
`694+-7'4+-{[]+<+<7+[+[]<[[[]`']2{{<+[{1+[]-`{<{<>[[`{}{}']]}[[`{[`']}']]>}'}]->}}]>-]->->-}
{{{`'}3}}{{}{`'<`'5>}{}{}<``<>[+-]``{`'+<`'4[{{}{}{}{``[+[]+-`'{`{{{}}}{<>+[]-}'}-]''}}]>-}''''>}
{}{`42+-[[{[]}{{{{}}}9}{{[{}+[<1`'{}`'2>[<>]5`'`{[8{{+{+`'<>`'5-}<`{}[{+-}]'>-}}]}']-]}}]]'}
[{[{6+{{}}[{{}}`'+-]<{}><{{<{}[]+{[]}[48{+[9<<>++<><>{[{0<>[`'{4}{}[`{}']]}]}-->]-}]->}}>-}]}]
{[]<0`'{{{{[<{{}`<{{[<+{}-`+-{}``''8{`[[`<>4`'[{{<<>`{}'[]<+{}`<>'3+-->>}}]']]'}'>]}}>'}>]}}}}>}
{{+{{9{{{{[{[{}][0[]<{0`3{`'[{`+{1}`<[2+{{}[+`0[{9++<`'>+-0--}]'-]}-]>'-'}]}'}>]}]}}}}}}-}}
{+-{{}{<<{}[<[``'<[]{{``''`{{+-+-}}[<>]9[`'{<>{}[]+[3[]``'']{{<{<{9[]<>`'[]}>}>}}-}]'}}>']>]>>}}}
`'{+<7{+[]-<<{[`<>{{+{}`'-}}'{{[]++-<{{`+-'}{}}+[]{`'}-{+4[{{{{<>}}{{+[]{{0}}-}}}}]-}>-}}]}>>}>-}
+-7{}<>{}{++`[[]{}]{[]+{{}[{+-+<`{3`{{+4{}++--{}`3+-[<{+<><[]>{}`<{+{}-}>'-}>]'-}}'}'>-}]}-}'--}
{[+-++{}4[{<`'>{}++<<><>[]>[+{{[{5{}+`[`+`'{+-{<[]+->{}}[09{[6{<>{{{}}}}]}]}-']'-}]}}-]--}]--]}
3[{{}<{}>{`'`{{}}`[+[]{{{`{`{}00{{<+4[<>++-[[{}5`{{{{{}+{7+{}-``''}-}}}}']]-]->}}'}'}}}-]''}}]
{{}`<>+<>{}[]-6``'{{{+<{`4`'<[]{+<[][<[[{}`<9[]{<>}{+-}<<[]+{`<>`'+-{`'}'}->>>']]>]>-}>'}>-}}}''}
<>[+[]-]+9{`'{[[]]{}}<+`+{}{{}[4{{+{}{{[3{5[{+{}-}+-]}{+{}``+```{+-}<{{}}>'''-''-}]}}-}}]}-'->}-
<<{{}`{}{[98``'']}<[]`{`<+`'->`'{}'<+-[{}+{<>{`16{}`'<>'[{}`'<>]}}-[+`{{<{{`'+-}}>}}'-]]>}'>'}>>
+-{`+-{[]{<>}{{}}}[<>]{}<{[]}[++--]>'+[<>]{}[<{8<>{{`'<4{`{++-+{`<7+4``0'[]'5->'}--}'}>}}}>]-}
<<>>{+{}<>+{{[[{2<{+{}<>[{<>}+<`+`{{}}{{<[]>}}[{<{}>+<>{+-}9-3{}51{+<>-`+-'}}]'-'>-]-}>}]]}}--}
{}`1'<8+-<>5{}7{{``{}''`'{+{}`<+-{+{36<>`[]+-[0`<`'+-<<<+->`'[{}{{{`<>'{}}}}]>>>']'}-}>'-}}}>
[`<<6`++-[]{+{+<>+{<>}+[+-{`[`+-+<`3[]{{+-<`{+-4[{{``+{<5{70}>}-''}}]}'>}}'>-']'}]---}-}-'>>']
{<>}<>`{{+-`'{<`<++-`<[][<>`{}<+-`7'+``'4{`+{{<`<[2]`{{3+-+-<[03<>]>}}'>'>}}-'}'->']>'->'>}}}'
+-58{{}[{`{{0[<>]+-6}}<>`+`8{<>{++[]``'{}{}{}{}9<>{{{[]`'+<>{}{}-}}{6[+<{{}}>-]}}'--}}'-''}]}
`[{6[+{{{}1+-<>}<`{3++{}2<{`{{{}}<{`[8]'}`[<4<>{{}}61+{2<{}[]>}[<{{}}>]->]'>}'}>--}'>}-]}]'
{[{+-{<6{```[]`{69{[+{<4+-[]{<>}{[{5<+-7[[{{9}[][]<{}{`+-'++-`{+-}'-}>}]]>}]}>}-]}}''''}>}}]}
`[]<>3``{[]}+[]`{[{8[]<>{}``{{`7[2{}]'`'[`6[]0[]{}0{[<>++[]-{<+<5`'0{}>->}-]}']}}''}]}'-'''
`'{}<`3{}[`{+-<<>{}>``[++-<[][+-{}]{[`'{+-[]+-5{[]}0<>`{}<{+-}>{{[[`{5{`'}`'}']]}}'}]}>-]''}']'>
{{<>{[{}]`{{1`'}`'`[]'}<+-`'{`+-'<`[3[]]'<>4{{}+-`1`{9}'[`+{{{86{{+-++{+-}--}}}}}-']'}>}>'}}}
`{+<{}{<>{}`'{}[]}>+-5<<>>345`{}+-'+<3`<[{<>{+-}[{[]`{}'{}[]`{{}{[][<>]}{[4`'`']}}'}]}]>'>--}'
<[]{[]{{`<{<{{{2{[[<``'`''[{{+-{`[{`<24[]{}<92+`3{{`2[+-[{{}}]]'}}'->>'}]'}}}]>]]}}}}>}>'}}}>
{[`{9}`6`'[]{`9`+[<`<`'{<`'>}>[{+[]-`{+[{`+[+-+{{22`0+[<>+{}-{{`'}}]-'}}-]-'}]-}'}]'>]-''}'']}
<>`+-{5{}`'`'{}<{[[[++--]<>`'{{}+-}]]}>{++2`'++--{{{2+<>{+8{[<7>]{{}<<>[<{{[]}}>]>}}-}-}}}--}}'
`{{8{`'7{}{[{+{}``+-2{}'`'{{`'`{}`'``'+[]<><>{[]}-'<{}[[{{}{<{`'[8{}{{`[]'}}]}>}}]]>'}}'-}]}}}}'
[<+[`'[{{}`[[{<>{}`{}+`'{8{}{[{}]}`{{+16<>[64]`'{<{{{{{{`<`[{}{}{}]'>'}}}}}}>}-}}'}-'}]]'}]]->]
<++8`+6{}`5+-'`[107{}{{`{}`<<{[<[]5{9{{`'}`'<{}[{}]{{}{+<>{0[{}{<>}]}-}}>}}>]}>>''}}]'-'-->
``++--'{}3+6{`{}'{`'{`'<<``{<<<<{}[]<[{`<{{`'}{<<9><>{{`'}{{}`'}}{`{{}}`''}>}}>'}]>>>>>}''>>}}}-'
<>[]3``'{}'`'{{{`'{<{}[+8+-`++{}{{3{{<{{{<[]{}>[]<>9`[{}<{`'<{[{{}[]`'}]}>}>]'}}}>}}}}--'-]>}}}}
{9}4{<<>+`{}[]'{}+{}-0{[{}<>`+[{}{{}}{}][{{++-+{}--}{1[{5[{{}{[]{<41{}4[`']>}}}]}]}}]-']}->}
+[6{}3{{[+<`'><>[]+0{[]}--]{[]}}9[{[{`'}][]}]+20`[]{{[2`'{[[{}{`+8-{}`'[[`'+-]]'}]]}]}}'-}]-
`<[+-`{{`{{{+<{+-}0``{+-{}<{<`'`<{+{{+-}[+-]{`'{7<<[]{}<{<>}81`6{}'>>>}}}-}>'>}>}''>-}}}'}}']>'
+[4]-`[]`[]1`'<>+<{+{``6`'[{{}}[[]+-[]`']{{`'}}]`[`'{+-[{{}}]{{5{[[]]<[{2{}{}}]>}}}}]'''}-}>-''
{{}{+[`+-9'`[<<{[[]<{}+[]{}[{[++[]-<3>[]+6<{++<`'+->+-2{`<`'>{`[]'`'`{}'}'}--}>--]}]->]}>>]']-}}
{}{}<>[][`[`'{[7<{8`1'}>+`'{``2`2[7{{{5{6[[+<>`'-]0{`'`{}{}3+<+-[{[]}]>-'}]}}}}]'''}-]}]']
`{`{}'}[]<`'+-+-[+{}[[<>]+[<>]+-<+{`+{}9{{<{+{<<[[`[]{[<[[]5[6<>5``'']]>]}']]>>}-}>}}-'}->-]-]>'
+-[5{<>+2<+-`[]'>[+9{{}}4[{++[+-{[`[`'`']{}{{+`8'5{`'}`3+`'[[[++{`+-'}--]]]-'-}}']}]--}]-]-}]
2`'+-{{}}{+71+-[+-]-}<[]>{<+{}{++`<{{``[]+`'[{}<{`{}'+{}{{+-[{[]}]{}`+<`{}'>-'}}-}>]-''}}>'--}->}
{{}<<+`[]<+-<>`{{}}+++[]<{{`+-{{}[+-]{+`{}{}`[{[0`[{[<`<{}[[+-]<>{}7]>'>]}]']}]''-}}'}}>---'>'->>}
<`'{}<<><>>`<>`'`'<9[3{{+-{`'`{1+-``''<{}{[]}`<{<>}6>{}+{[+<[{`{[[`']]}'[{}]{}}]>-]}-'>}'}}}]>'>
{{{}{8}2{`'}+{[`'{{{}}}]}{}<[2{{}}54{+`<<>4+-[]{}{}>{}```+`'-++{{}[]}--`[`{{<>}}']'''''-}]>-}}
{}{}`+1-<>{}+{}{{{{}`+-'`[+-3{}{[[]]+{<>[{+-[{{}[{+-}<>+`'{+-{9}{+-[][]++-`+-'-}}-]}]}]}-}]'}}}-'
