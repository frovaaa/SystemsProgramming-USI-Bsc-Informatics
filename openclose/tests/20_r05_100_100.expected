<+{+<3{[`{`++`{4<<{`5[0<{[9]}>]'}>>}'--'}']}>-}->
[9{<`+0-[3]{`<64`{{6}[``{<{`743956'}>}'']}'>'}'>}]
`<{`6'`4<<{{8`[8<[++`0+<{5+655[[[{{6}}]]]-}>-'--]>]'}}>>'}>'
+{{++`<{`+<+[<5{+7`{+[{`<{5{<<4>>}}>'}]-}'-}>]->-'}>'--}}-
6[0`{{{{[{<`<{5{+{4{782[{<++[++`2'--]-->}]}}-}}>'>}]}}}}']
+<`[9+[[<{[5]}>{{{`{1{<+<<17>>->1}}'}}}]]-]'>-
{`<{<[<{<`{5}'>{{{<6>{[[`{{<+`82{3}'->}}']]}}}}}>]>}>'}
[+[4<[[9`7{{`9{<{``{3}''}>}'}}']]>]-]
+{{9{{<<{<+<{{[6]}}+<{9}>->->}>>}}}}-
<{47+{{`62{{+9`++4--'-}}'}}-}>
<7[{{9<<`1{`[<<7>>]'}'>>}}]>
0++{`{{2[14]}<<{5}>>}'}--
`[{{{[+[{+`{{`<{``<5>{2{<[{{9+[{4<`+`{0}'-'>}]-}}]>}}''}>'}}'-}]-]}}}]'
[[9[+`<`{{{<{<0>}>}}}'>'-]]]
``+8`4'+[[{{{{[{{+{[[+[7]-]]}-}}]}}}}]]--''
[`[+`{{{[[8{`9'}{[`{1}[++6`+2+<0>--'--]']}]]}}}'-]']
<<5>>`3<[`[4[<0+{<`7{<[[[+`<{[`<{1}>']}>'-]]]>}'>}->]]']>'
4<<9[5{{+{{<`[{52}]'>}}-}}]>>
{69{{{1`{{{[<<48`{<61>}[7<`8+9-'>]'>>]}}}'}}}}
`<[`[<{{+3`{{{2<{[{[[<{<[<`3'>]>}>]]}]}>}}}'-}}>]']>'
<8+<+4[+`1{[`[2{{<{1{{<+{{<{[6]}>}}->}}}>}}]']}'-]->->
`<2>``{{2}}76{+`+[<<{{[[<<{[2`+`38{<{2}>}'-']}>>]]}}>>]-'-}'''
{1++8<58{{1<[`+[<`{6}'>]-']>}}>--}
<++[0{4{<{5}>}{9}}]-->
++2{+{<<11[`+`<`[6{{2}}]'>'-']>>}-}--
`9+6-'<`9<<{{`5'`<[<+`+`7<8<0>>'-'->]>'}}>>'>
`[{`65'+`2'++``{{84`++{<7>{6}}--'}}''---}]'
3[[{{{+2+0<{``4[+<+7`<5>{8{7}}'->-]''}>--}}}]]
`{`<<{[[<<`4`<`+`{`5{{6}}'}'-'>''>>]]}>>'}'
+7{1+{0[{`[4]'}]}-}-

`<{[+++{<[[5+{{{{+5<<+7->3>-}}}}-]]>}---]}>'
{{{<<<5{[+[+8-[+<3{{+{{<+9->}}-}}>-]]-]}>>>}}}
{3[+++<{6<+`[+[[[`<`[9{{7}}+2-]'>']]]-]'->}>---]}
<+<+{[6+[{+4-}]-6]}+[5+4+++{{7}}----]-->->
3<[`[[[{{{[{+{+[<<[<<9>>`[0{+8-}]']>>]-}-}]}}}]]]']>
+`6[+`{2}'-][`[{<1{{5<+[++`+`<2>'-'--]->}}>}]']'-
{{<7`+`{`+{[<`{<{2}>}'>]}-'}'-'>}}
+`1<1<{<`{`<{<`7<{5{<{{`{{{9}}}'}}>}}>'>}>'}'>}>>'-
`{4[`+`<+`{+<[1+2-]>-}'->'-']}'
<[`<{[+<8{<{+[+1{9{`++`3'--'}}-]-}>}>-]}>']>
[{6{{3}6[<`<<6[<{2}{<4>}>{{{+[7]-}}}]>>'>]}}]
+9{{`[[+[<[[``6{{<<+{{7{6}}}->>}}'']]>]-]]'}}-
+``96``+[[<`[{{{{+{[{0[[{1}]]}]}-}}}}]'>]]-''''-
{{`{[+{{{`[+`{<<`<<{{{4}}}>>'>>}'-]'}}}-]}'}}
5[[<++[{{{07{{+{5<`[87<<<[[2]]>>>]'>}-}}}}}]-->]]
<{+{<{+`<<+[4{[+<+3->-]}]->>'-}>}-}>
{``{`0'}2{<{<[{[1[`{{{4{<9<+[+{{<3>}}-]->>}}}}']]}]>}>}''}

{{+[<{8<2+<[`<``+<3{<``<83{<{<`+{{[1]}}-'>}>}>''>}>-''>']>->}>]-}}
[+{2{`9{[4]{7}<{{{`{`{5}'}'}}}>}'}}-]
<`[[<5<[[{{``+{<[`0'+{[3]}-]>}-''}}]]>>]]'>
{[827[+`<[+{{<{+<{`<7>'}>-}>}}-]>'-]]}
`6947'4{+<+<7+[+<[2{{<+[{1}]->}}]>-]->->-}
{{3}}{{<5>}<````{+<4>-}''''>}
{`42[[{9}{{[+[<12>5`{[8{{+{+5-}-}}]}']-]}}]]'}
[{[{6+<{{<+[48{+[9<++{[{0[{4}]}]}-->]-}]->}}>-}]}]
{<0{{{{[<{`<{{[<`8{`[[`4[{{<<+3->>}}]']]'}'>]}}>'}>]}}}}>}
{{+{{9{{{{[{[0<{0`3{[{`+{1}`<[2+{[+`0[{9++0--}]'-]}-]>'-'}]}'}>]}]}}}}}}-}}
{{{<<[<[`<{{`9[{+[3]{{<{<{9}>}>}}-}]'}}>']>]>>}}}
{+<7{<<{[{{+<{+4[{{{{+{{0}}-}}}}]-}>-}}]}>>}>-}
7{++`{+{[{+<`{3`{{+4`3'-}}'}'>-}]}-}'--}
{[++4[{++[+{{[{5+`[`+{[09{[6]}]}-']'-}]}}-]--}]--]}
3[{{``[+{{{`{`00{{<+4[+[[5`{{{{+{7}-}}}}']]-]->}}'}'}}}-]''}}]
{`6`{{{+<{`4<{+<[<[[`<9>']]>]>-}>'}>-}}}''}
+9{<+`+{[4{{+{{[3{5}]}}-}}]}-'->}-
<<{`{[98]}<`{<[+{{`16'}}-]>}'>'}>>
{+[<{8{{<4{`{++{`<7+4``0''5->'}--}'}>}}}>]-}
{++{{[[{2<{+[+<`+`[{+9-351}]'-'>-]-}>}]]}}--}
`1'<857{{{+`<{+{36`[0]'}-}>'-}}}>
[`<<6`+{+{+++[{`[`+<`3{{<`{4[{{``+{<5{70}>}-''}}]}'>}}'>-']'}]---}-}-'>>']
`{{{<`<+`<[`<`7'+`4{`+{{<`<[2]`{{3<[03]>}}'>'>}}-'}'->']>'->'>}}}'
58{[{`{{06}}`+`8{{++`9{{6}}'--}}'-''}]}
`[{6[+{{1}<`{3++2<{`{<{`[8]'}`[<461+{2}->]'>}'}>--}'>}-]}]'
{[{{<6{````{69{[+{<4{[{5<7[[{{9}}]]>}]}>}-]}}''''}>}}]}
`3``+`{[{8``{{`7[2]'[`600{[+{<+<50>->}-]}']}}''}]}'-'''
<`3[`{``[+<{[{50`{{[[`{5}']]}}'}]}>-]''}']'>
{{{`{{1}}<{<`[3]'4{`1`{9}'[`+{{{86}}}-']'}>}>'}}}
`{+5345+<3`<[{[{`{{[4]}}'}]}]>'>--}'
<{{{`<{<{{{2{[[<[{{{`[{`<24<92+`3{{`2'}}'->>'}]'}}}]>]]}}}}>}>'}}}>
{[`{9}`6{`9`+[<`[{`{+[{`+[+{{22`0'}}-]-'}]-}'}]'>]-''}'']}
`{5{++2{{{2+{+8{[<7>]}-}-}}}--}}'
`{{8{7{[{+``2'{{`<[[{{<{[8]}>}}]]>'}}'-}]}}}}'
[<+[[{`[[{`+{8`{{+16[64]-}}'}-'}]]'}]]->]
<++8`+6`5'`[107{{``<<{[<5{9{<{{+{0}-}}>}}>]}>>''}}]'-'-->
`3+6{{{<<``{<<<<<[{`<{{<<9>>}}>'}]>>>>>}''>>}}}-'
3{{{{<[+8`++{{3{{<{{{9}}}>}}}}--'-]>}}}}
{9}4{<+0{[`+[{{1[{5[{{{<414>}}}]}]}}]-']}->}
+[63{{[++0--]}9+20`{{[2{[[{`+8-'}]]}]}}'-}]-
`<[`{{`{{{+<0``{<{<`<{+{{{7<<<81`6'>>>}}}-}>'>}>}''>-}}}'}}']>'
+[4]-``1+<{+{``6`[{{{5{<[{2}]>}}}}]'''}-}>-''
{{+[`9'`[<<{[<+[{[+<3>+6<{++2--}>--]}]->]}>>]']-}}
[`[{[7<{8`1'}>+{``2`2[7{{{5{6[0{`3'}]}}}}]'''}-]}]']
`<[+[+<+{`+9{{<{+{<<[[`{[<[5[65]]>]}']]>>}-}>}}-'}->-]-]>'
[5{+2[+94[{++[{[`{{+`8'5`3'-}}']}]--}]-]-}]
2{+71-}
{<<+`<`+++<{{`{{+``[{[0`[{[<`<[7]>'>]}]']}]''-}}'}}>---'>'->>}
<`<9[3{{{`{1<`<6>'>}'}}}]>'>
{{{8}2+<[254{+`<4>'-}]>-}}
`+1-+{{{`[3{+{[{[{[+{{9}}-]}]}]}-}]'}}}-'
